src/status_im/contexts/wallet/add_account/create_account/new_keypair/confirm_backup/view.cljs

```diff
@@ -118,23 +118,23 @@
 
 (defn view
   []
-  (let [random-indices         (random-selection)
-        quiz-index             (reagent/atom 0)
-        incorrect-count        (reagent/atom 0)
-        show-error?            (reagent/atom false)
-        {:keys [on-success
-                on-try-again
-                masked-seed-phrase
-                back-button?
-                theme shell?]} (quo.context/use-screen-params)
```

Comment:
Thank you, @ulisesmac, for reporting this issue. For some reason, only this use case was triggering a warning. However, as a precautionary measure, I have migrated all other uses as well.